lib/client/getUniqIdForGroupedIssues.ts

```diff
@@ -6,7 +6,14 @@ interface GetUniqIdForGroupedIssuesArgs extends Omit<DetailedViewGroup, 'items'>
 }
 export default function getUniqIdForGroupedIssues (groupedIssues: GetUniqIdForGroupedIssuesArgs[]): string {
   return groupedIssues.map((group) => {
-    const groupChildrenId = group.items.map(({ node_id }) => node_id).join('-')
+    const groupChildrenId = group.items.map((item) => {
+      try {
+        return item.node_id;
+      } catch (e) {
+        console.error(e)
+        return '';
```

Comment:
Can we return a uniqId for the item instead of an empty string? an empty string where there are failures could cause issues. Also, can we log the item with the issue as well?
```suggestion
        console.error('Problem getting node_id for group item:', item)
        console.error(e)
        try {
            return JSON.stringify(item);
        } catch {
            return ''
        }
```

we should create a uniqId function for `IssueData` types that handles missing fields.. but we should also figure out why this function is being called without a typeError with some `.items` that are missing a node_id property..